.github/workflows/sam_release_code.yml

```diff
@@ -156,3 +158,37 @@ jobs:
           # TRUSTSTORE_FILE: ${{ inputs.TRUSTSTORE_FILE }}
           VERSION_NUMBER: ${{ inputs.VERSION_NUMBER }}
         run: ./release_code.sh
+
+      - name: Set ECR Repository and Image Tag
+        run: |
+          echo "ECR_REPOSITORY=fhir-facade-repo" >> "$GITHUB_ENV"
+          echo "IMAGE_TAG=latest" >> "$GITHUB_ENV"
```

Comment:
Image tag for the pull request should be PR-pullRequestId. 

Image tag for merges to main should be the short commit hash and version number.